src/components/searchResult/GoldenSearchResult.tsx

```diff
@@ -0,0 +1,323 @@
+import { MoreHorizOutlined } from '@mui/icons-material'
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp'
+import SearchIcon from '@mui/icons-material/Search'
+import {
+  Box,
+  Container,
+  Grid,
+  Link,
+  Paper,
+  TableSortLabel
+} from '@mui/material'
+import Collapse from '@mui/material/Collapse'
+import Divider from '@mui/material/Divider'
+import IconButton from '@mui/material/IconButton'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TablePagination from '@mui/material/TablePagination'
+import TableRow from '@mui/material/TableRow'
+import Typography from '@mui/material/Typography'
+import { visuallyHidden } from '@mui/utils'
+import { MakeGenerics, useSearch } from '@tanstack/react-location'
+import { useQuery } from '@tanstack/react-query'
+import { AxiosError } from 'axios'
+import React from 'react'
+import { useAppConfig } from '../../hooks/useAppConfig'
+import ApiClient from '../../services/ApiClient'
+import { DisplayField } from '../../types/Fields'
+import {
+  CustomGoldenRecord,
+  Data,
+  patientRecord
+} from '../../types/SearchResults'
+import { SearchQuery } from '../../types/SimpleSearch'
+import Loading from '../common/Loading'
+import PageHeader from '../shell/PageHeader'
+
+interface EnhancedTableProps {
+  onRequestSort: (
+    event: React.MouseEvent<unknown>,
+    property: keyof CustomGoldenRecord
+  ) => void
+  sortAsc: boolean
+  sortBy: string
+  rowCount: number
+  availableFields: DisplayField[]
+}
+
+interface HeadCell {
+  id: keyof CustomGoldenRecord
+  label: string
+}
+
+type ResultProps = MakeGenerics<{
+  Search: {
+    payload: SearchQuery
+  }
+}>
+
+type Order = 'asc' | 'desc'
+
+const headCells: readonly HeadCell[] = [
+  {
+    id: 'uid',
+    label: 'Golden ID'
+  },
+  {
+    id: 'givenName',
+    label: 'First Name'
+  },
+  {
+    id: 'familyName',
+    label: 'Last Name'
+  },
+  {
+    id: 'gender',
+    label: 'Genre'
+  }
```

Comment:
Same here, we need to get the head cells using the available fields.

Reply:
we actually dont need it. I forgot them. Thanks